### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by Chithrak07**

### CODE REVIEW

**Comment by Chithrak07** [comment]
@dale-c-anderson : This runs ok in local

File: molecule/default/converge.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - make_cmd: 'make -s install'
+
+  pre_tasks:
+
+    - name: Update apt cache.
+      apt: update_cache=true cache_valid_time=600
+      changed_when: false
+
+    - name: "Installing Rsync"
+      apt: >
```
Comment (dale-c-anderson) [inline molecule/default/converge.yml]: Same as all the rest of the items that now test centos - apt is ubuntu/debian-specific.

Comment (Chithrak07) [reply molecule/default/converge.yml]: Fixed those.

This runs ok in local. But throws the "rsync" error in github.

**Review by dale-c-anderson — APPROVED** [review]
Reviewed

**Pull request merged**